<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZoomRotateDialog</class>
 <widget class="QDialog" name="ZoomRotateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>214</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lantinghei SC</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>缩放/旋转</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QRadioButton
{
	color:rgb(221, 221, 221);
}
</string>
  </property>
  <widget class="QSlider" name="zoomSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="rotateSlider">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="conformButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="zoom">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>33</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>缩放：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rotate">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>83</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>旋转：</string>
   </property>
  </widget>
  <widget class="QLabel" name="zoomValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>33</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="rotateValue">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>83</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="nearest">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>最近邻</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="linear">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>双线性</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
